airmozilla/main/models.py

```diff
@@ -20,7 +20,7 @@
 
 
 def _get_now():
-    return timezone.now()
+    return datetime.datetime.utcnow().replace(tzinfo=utc)
```

Comment:
This change, and the change above must have been a little mistake when you rebased or something. The correct code should be:
```python
from django.utils import timezone
...
def _get_now():
    return timezone.now()
```